src/main/java/org/folio/fqm/service/EntityTypeService.java

```diff
@@ -67,9 +70,17 @@ public List<EntityTypeSummary> getEntityTypeSummary(Set<UUID> entityTypeIds) {
   public Optional<EntityType> getEntityTypeDefinition(UUID entityTypeId) {
     return entityTypeRepository
       .getEntityTypeDefinition(entityTypeId)
-      .map(localizationService::localizeEntityType);
+      .map(localizationService::localizeEntityType)
+      .map(this::sortColumnsInEntityType);
   }
 
+  private EntityType sortColumnsInEntityType(EntityType entityType) {
+    List<EntityTypeColumn> sortedColumns = entityType.getColumns().stream()
+      .sorted(Comparator.comparing(EntityTypeColumn::getName))
+      .collect(Collectors.toList());
+    entityType.setColumns(sortedColumns);
+    return entityType;
+  }
```

Comment:
Since this modifies its parameter object, I wouldn't return anything to make it obvious that this has side effects (`void` methods _always_ have side effects, or they do nothing at all, so it's a good indicator). Alternatively, this could create a new `EntityType` and return that (which I'd actually prefer since it's easier to reason about, but it's a lot more work/code)

Reply:
If we use Lombok `@With` on the entity, you can just use `.withColumns` instead